<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ventana_login</class>
 <widget class="QDialog" name="Ventana_login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>489</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #f0f0f0</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>260</y>
     <width>271</width>
     <height>51</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>271</width>
     <height>51</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="accessibleName">
    <string/>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #4CAF50;
font: 14pt &quot;MS Shell Dlg 2&quot;;
color: white;
border-radius: 8px</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="campo_usuario">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>90</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> border: 2px solid rgba(255, 255, 255, 0.9);  
    background-color: rgba(255, 255, 255, 0.6);   
    padding: 5px; 
    selection-background-color: #3498db;  
font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="campo_pass">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>150</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> border: 2px solid rgba(255, 255, 255, 0.9);  
    background-color: rgba(255, 255, 255, 0.6);   
    padding: 5px; 
    selection-background-color: #3498db;  
font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::CustomContextMenu</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"> border: 2px solid rgba(255, 255, 255, 0.9);
font: 75 15pt &quot;MS Sans Serif&quot;;
font-size: 18px; 
font-weight: bold; 
text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.5);
background-color: rgba(255, 255, 255, 0.5);   </string>
   </property>
   <property name="text">
    <string>Usuario:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> border: 2px solid rgba(255, 255, 255, 0.9);
font: 75 15pt &quot;MS Sans Serif&quot;;
font-size: 18px; 
font-weight: bold; 
text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.5);
background-color: rgba(255, 255, 255, 0.5);   </string>
   </property>
   <property name="text">
    <string>Contrase√±a:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_imagen">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>-10</y>
     <width>500</width>
     <height>500</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>Imagenes/login.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <zorder>label_imagen</zorder>
  <zorder>buttonBox</zorder>
  <zorder>campo_usuario</zorder>
  <zorder>campo_pass</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
